great-idea/index.html

```diff
@@ -17,7 +18,72 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
-      
+       <nav>
+      <ul>
+        <li><a href="services.html"></a>Services</li>
+        <li>Product</li>
+        <li>Vision</li>
+        <li>Features</li>
+        <li>About</li>
+        <li>Contact</li>
+        <li><img class="logo" src="img/logo.png" alt="Great Idea! Companylogo."></li>
+      </ul>
+    </nav>
+
+    <header>
+       
+        <h1>
+          Innovation
+          <br>On
+          <br>Demand
+          <br>
+          <br>
+          <div class="started">Get Started</div>
+        </h1>
+      <img class="header" src="img/header-img.png" alt="Image of a code snippet.">
+
+    </header>
+
+    <section class="two-columns">
+      <h3>Features</h3>
+      <p class="left">Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+
+      <h3>About</h3>
+      <p class="right">Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </section>
+
+    
+      <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    
+
+    <section class="three-columns">
+      <h3>Services</h3>
+      <p class="left">Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+
+      <h3>Product</h3>
+      <p class="left">Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+
+      <h3 class="left">Vision</h3>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </section>
+
+    <section class="contact">
+      <br>
```

Comment:
No need to do a `<br>` here unless you wanted the extra spacing. Generally, you would remove this and adjust in the CSS file.